var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -43,7 +43,6 @@ class Pango(AutotoolsPackage):
 
     depends_on("pkg-config", type="build")
     depends_on("harfbuzz")
-    depends_on("cairo")
```

Comment:
What does your `packages.yaml` look like? With no `packages.yaml`, it should concretize correctly. I made sure that the `X` variant for every package was disabled by default, and the `mpi` variant is enabled for every package by default. If you change these defaults in your `packages.yaml` or on the command line, you're screwed unless you change the dependencies too. Hence the broken concretizer.